src/lib/features/listening-history/model/specs/store.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { addToListeningHistory, listeningHistory } from '../store';
+import { get } from 'svelte/store';
+import type { Episode } from '$lib/shared/api';
+
+const exampleEpisode: Episode = {
+  id: 1,
+  episodeNumber: 1,
+  podcastID: 1,
+  title: 'Episode 1',
+  audioUrl: 'http://example.com',
+  duration: 5,
+};
+
+const expectedHistory: number[] = [exampleEpisode.id];
+
+it('listen to an episode', () => {
```

Comment:
suggestion (non-blocking): the code reads better when using -s forms:
```suggestion
it('listens to an episode', () => {
```

but then
question: how does this test that the code "listens to an episode"?